### PR TITLE
Feedback for 'About' page

### PR DESCRIPTION
**Pull request by sfmig**
Fixes small typos and adds some suggestions to make the about page more readable. 

Mainly some rephrasing to avoid repetition of some words and makes some sentences a bit shorter. Of course it can be quite subjective so feel free to take it or leave it!

### General comments
- I find it a bit confusing that in the rendering of the page there are two navigation panels, on the right and left hand side of the main page.
- Potentially personal, but is it possible to make all links open in a new tab by default?

### CODE REVIEW

**Comment by adamltyson** [comment]
@sfmig Would you mind rebasing this onto #31? I think we'll end up with merge conflicts. 

> I find it a bit confusing that in the rendering of the page there are two navigation panels, on the right and left hand side of the main page.

TBH I've never noticed it, but now you mention it, it is a bit weird. @niksirbi I think we have this on all our pages. Can we change this? Should we change this?

> Potentially personal, but is it possible to make all links open in a new tab by default?

Not by default I don't think, but it can be done for external links. I'll raise an issue.

**Comment by adamltyson** [comment]
> Not by default I don't think, but it can be done for external links. I'll raise an issue.

Actually this already happens I think. I'm not sure we want this to happen for all links, otherwise it will get messy quickly. From the selection of tech docs I had open (zarr, pandas, numpy), other tools don't do it, so I wouldn't add this unless there's a particular reason to. Any thoughts @brainglobe/czi_eoss5

**Comment by dstansby** [comment]
I'd say it's pretty standard practice not to open internal links in a new window.

**Comment by niksirbi** [comment]
> TBH I've never noticed it, but now you mention it, it is a bit weird. @niksirbi I think we have this on all our pages. Can we change this? Should we change this?

This is the [default setting of the Pydata Sphinx theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/layout.html), and one of its core features/characteristics. It can be removed globally or on particular pages, but I'm not sure we want to do that. Other big projects that use the same theme for their docs, e.g [numpy](https://numpy.org/doc/stable/index.html) and [pandas](https://pandas.pydata.org/docs/), stick to the default behaviour with the two sidebars - left for navigation across documents/pages, right for within-page navigation (especially useful for long markdown files).

File: docs/source/about.md [hunk 1]
```diff
@@ -6,34 +6,32 @@ The BrainGlobe Initiative (BGI) was established by researchers at the
 Technical University of Munich](https://www.tum.de/en/). 
 
 The core goal of BrainGlobe is to develop a suite of Python-based software tools for computational neuroanatomy. 
-We have developed several software packages to access, analyze and visualize anatomical data. By ensuring the 
-interoperability of all of BrainGlobe's software we aim to streamline the development of analysis pipelines and 
-facilitate the process of going from raw data to publication-ready content. By producing a set of high-quality 
+We have already developed several software packages to access, analyze and visualize anatomical data. By ensuring the
```
Comment (adamltyson) [inline docs/source/about.md:9]: Not sure why, but I don't like the "already"
```suggestion
We have developed several software packages to access, analyze and visualize anatomical data. By ensuring the 
```

File: docs/source/contact.md [hunk 2]
```diff
@@ -24,5 +24,5 @@ but please take a quick look at the [developer guide](developers/index) first.
 
 ## Other ways to get in touch
 While we prefer that all discussions about the project are carried out in public, there may be situations in which this 
-is not possible. In these situations, please [email](mailto:code@hello@brainglobe.info?subject=brainglobe), the core developers.
+is not possible. In these situations, please [email](mailto:code@brainglobe.info?subject=brainglobe), the core developers.
```
Comment (adamltyson) [inline docs/source/contact.md:27]: Not sure how this email got so garbled. 
```suggestion
is not possible. In these situations, please [email](mailto:hello@brainglobe.info?subject=brainglobe) the core developers.
```

**Review by adamltyson — APPROVED** [review]

**Comment by adamltyson** [comment]
Thanks @sfmig! I made a couple of tiny changes.

**Pull request merged**